app/src/main/res/values/arrays.xml

```diff
@@ -7,6 +7,7 @@
         <item>Wifi Wixel + BT Wixel</item>
         <item>G4 Share Receiver</item>
         <item>G5 Transmitter</item>
+        <item>LimiTTer</item>
```

Comment:
Are we sure that the name that the user will see is LimiTTer?
Is there anyway to connect it with the libere?